cmd/exec.go

```diff
@@ -0,0 +1,47 @@
+package cmd
+
+import (
+	"context"
+	"os"
+
+	"github.com/fromanirh/pack8s/internal/pkg/podman"
+	"github.com/spf13/cobra"
+)
+
+type execOptions struct {
+	commands []string
+}
+
+var execOpt execOptions
+
+// NewExecCommand runs given command inside container
+func NewExecCommand() *cobra.Command {
+	exec := &cobra.Command{
+		Use:   "exec",
+		Short: "exec runs given command in container",
+		RunE:  execCommand,
+		Args:  cobra.MinimumNArgs(1),
+	}
+
+	exec.Flags().StringArrayVar(&execOpt.commands, "commands", []string{}, "commands which will be executed in container")
+
+	return exec
+}
+
+func execCommand(cmd *cobra.Command, args []string) error {
+	containerID := args[0]
+
+	podmanSocket, err := cmd.Flags().GetString("podman-socket")
+	if err != nil {
+		return err
+	}
+
+	ctx := context.Background()
+
+	hnd, err := podman.NewHandle(ctx, podmanSocket)
+	if err != nil {
+		return err
+	}
+
+	return hnd.Exec(containerID, execOpt.commands, os.Stdout)
```

Comment:
now that I see how you want to use this, I think `--commands` is bad UX.
First, it should be `--command` (singular), because `exec` can run just a single command - not more than one.
Second, we should not require it to be comma separated. It should be a regular shell line. The simplest way is probably to pass through `args` from line 31 in hnd.Exec() here on line 46, perhaps with some minimal massaging.
This can improved later, but we should have it before submitting the kubevirtci PR.